app/main.py

```diff
@@ -1,52 +1,39 @@
-import json
-import xml.etree.ElementTree as ET
+from app.book import Book
+from app.book_display_classes import (
+    BookConsoleDisplayer,
+    BookReverseDisplayer,
+)
+from app.book_print_classes import BookConsolePrinter, BookReversePrinter
+from app.book_serialize_classes import BookSerializerJSON, BookSerializerXML
 
 
-class Book:
-    def __init__(self, title: str, content: str):
-        self.title = title
-        self.content = content
+def main(book: Book, commands: list[tuple[str, str]]) -> None | str:
+    display_methods = ["console", "reverse"]
+    print_methods = ["console", "reverse"]
+    serialize_methods = ["json", "xml"]
+    all_method_types = (display_methods + print_methods + serialize_methods)
 
-    def display(self, display_type: str) -> None:
-        if display_type == "console":
-            print(self.content)
-        elif display_type == "reverse":
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown display type: {display_type}")
-
-    def print_book(self, print_type: str) -> None:
-        if print_type == "console":
-            print(f"Printing the book: {self.title}...")
-            print(self.content)
-        elif print_type == "reverse":
-            print(f"Printing the book in reverse: {self.title}...")
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown print type: {print_type}")
-
-    def serialize(self, serialize_type: str) -> str:
-        if serialize_type == "json":
-            return json.dumps({"title": self.title, "content": self.content})
-        elif serialize_type == "xml":
-            root = ET.Element("book")
-            title = ET.SubElement(root, "title")
-            title.text = self.title
-            content = ET.SubElement(root, "content")
-            content.text = self.content
-            return ET.tostring(root, encoding="unicode")
-        else:
-            raise ValueError(f"Unknown serialize type: {serialize_type}")
+    for cmd, method_type in commands:
 
+        if method_type in all_method_types:
 
-def main(book: Book, commands: list[tuple[str, str]]) -> None | str:
-    for cmd, method_type in commands:
-        if cmd == "display":
-            book.display(method_type)
-        elif cmd == "print":
-            book.print_book(method_type)
-        elif cmd == "serialize":
-            return book.serialize(method_type)
+            if cmd == "display":
+                if method_type == "console":
+                    BookConsoleDisplayer.display(book)
+                BookReverseDisplayer.display(book)
```

Comment:
Create dict for this
displays = {
     "console": BookConsoleDisplayer,
     "reverse": BookReverseDisplayer
}
Code will looks more better
Also fix for other

Reply:
Thank you for your remark.